packages/sitecore-jss/src/native-fetcher.ts

```diff
@@ -111,6 +121,16 @@ export class NativeDataFetcher {
     }
   }
 
+  async fetchStream<T>(
+    url: string,
+    options: RequestInit = {}
+  ): Promise<NativeDataFetcherResponse<ReadableStream<T>>> {
+    const parseStreamResponse = (response: Response) => {
```

Comment:
how are we handling this in case of an error?

Reply:
there's no error to be had, we only return a value